.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+Please answer these questions before submitting your issue.
```

Comment:
In github is more common to use issue than bug-report. I would keep it as it is.